sycl/doc/extensions/supported/sycl_ext_oneapi_sub_group_mask.asciidoc

```diff
@@ -16,61 +17,71 @@
 // docbook uses c++ and html5 uses cpp.
 :language: {basebackend@docbook:c++:cpp}
 
-== Introduction
-IMPORTANT: This specification is a draft.
 
-NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by permission by Khronos.
+== Notice
 
-This document describes an extension which adds a `sub_group_mask` type.  Such a mask can be used to efficiently represent subsets of work-items in a sub-group for which a given Boolean condition holds.
+[%hardbreaks]
+Copyright (C) 2023-2023 Intel Corporation.  All rights reserved.
```

Comment:
```suggestion
Copyright (C) 2021-2023 Intel Corporation.  All rights reserved.
```

Reply:
Applied in 366c3ea5f81dbaec0d4429f593f6e74b72eec266